### PR TITLE
Update opensource.md

### PR DESCRIPTION
**Pull request by Eliyahou**
add Plagiarism-checker-Python to  slides

### CODE REVIEW

File: docker/opensource.md [hunk 1]
```diff
@@ -86,6 +86,15 @@ $ docker run -it --rm --workdir /opt -v$(pwd):/opt ubuntu:22.10 bash
...
 # phpunit
 ```
-
+## Plagiarism-checker-Python
+$ git clone https://github.com/Kalebu/Plagiarism-checker-Python
+$ cd Plagiarism-checker-Python
+$ docker run -it --name Renormalizer-devPlagiarism-checker-Python -w /opt -v %cd%:/opt python:latest bash
+$ pip3 install -r requirements.txt
+# running the project
+$ Plagiarism-checker-Python-> python3 app.py
```
Comment (szabgab) [inline docker/opensource.md:95]: What does this `Plagiarism-checker-Python-> ` prefix do here?
Did you manage to run the code this way?

File: docker/opensource.md [hunk 2]
```diff
@@ -86,6 +86,15 @@ $ docker run -it --rm --workdir /opt -v$(pwd):/opt ubuntu:22.10 bash
 # composer install
 # phpunit
 ```
-
+## Plagiarism-checker-Python
```
Comment (szabgab) [inline docker/opensource.md:89]: If you look at the rest of the file you'll see that each page also has an id beneath the title.

File: docker/opensource.md (hunk 1 above)
Comment (Eliyahou) [reply docker/opensource.md:95]: This is the name of the Directory
I just wanted that the prompt will have the same name as the Directory

Comment (szabgab) [reply docker/opensource.md:95]: Inside the docker container that is not the name of the directory and besides `$` was already there to indicated the prompt

Comment (Eliyahou) [reply docker/opensource.md:95]: it is the name of the directory and the name of the container
Screenshot 2023-05-04 114536

Comment (szabgab) [reply docker/opensource.md:95]: That's confusing here.

**Comment by szabgab** [comment]
I'll accept this PR and will fix it myself

**Pull request merged**